index.html

```diff
@@ -50,160 +50,168 @@ <h1>I'm Echezona <br>Glad to see <br>you!</h1>
+                <li><a href=""><img src="image/Vector.png" alt=""></a></li>
+                <li><a href=""><img src="image/Vector(3).png" alt=""></a></li>
             </ul>
         </div>
-        <button><a href="">Get my resume</a></button>
-        <div class="dropdown-menu">
-            <p>Languages</p>
-            <div class="drop1">
+        <button type="button"><a href="">Get my Resume</a></button>
+        <div>
+            <ul>
+                <li><p>Languages</p> <p>˅</p></li>
                 <ul>
-                    <li><a href=""><img src="image/Ellipse1.png" alt=""><p>JavaScript</p></a></li>
-                    <li><a href=""><img src="image/Ellipse1(2).png" alt=""><p>HTML</p></a></li>
-                    <li><a href=""><img src="image/Ellipse1(1).png" alt=""><p>CSS</p></a></li>
+                    <li><img src="image/Ellipse1.png" alt="">javaScript</li>
+                    <li><img src="image/Ellipse1(1).png" alt="">HTML</li>
+                    <li><img src="image/Ellipse1(2).png" alt="">CSS</li>
                 </ul>
-            </div>
-            <div class="drop2">
-                <p>Frameworks</p><img src="" alt="">
-                <p>Skills</p><img src="" alt="">
-            </div>
+                <li class="link2"><p>Frameworks</p> <p>></p></li>
+                <li class="link3"><p>Skills</p> <p>></p></li>
+            </ul>
         </div>
     </div>
 
 
-    <div class="form-box">
-        <div class="text">
-            <ul>
-                <li>Contact me</li>
-            </ul>
-            <ul>
-                <li>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </li>
-            </ul>
+    <div class="contact-container">
+        <div class="contact-me">
+            <h2>Contact me</h2>
         </div>
-        <div class="form-case">
-            <form action="">
-                <label for="">
-                    <input type="text" name="name" id="name" placeholder="Enter Name" required>
-                </label>
-                <label for="">
-                    <input type="email" name="email" id="email" placeholder="Enter Email" required>
-                </label>
-                <textarea name="" id="" cols="30" rows="10" placeholder="Write your message here"></textarea>
-                <button><a href="">Get in Touch</a></button>
-            </form>
+        <div class="para">
+            <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
         </div>
-        
+        <form action="https://formspree.io/f/{form_id}" method="post">
+            <div>
+                <ul>
+                    <li>
+                        <label for="name"></label>
+                        <input type="text" id="name" name="user-name" placeholder="Name" required>
```

Comment:
- [ ] Kindly make sure your input has some validation as required. The name is supposed to accept only up to 30 characters
`hint: You can do that by setting the maxlength attribute="30"`